hal/src/main/native/include/MockData/NotifyListenerVector.h

```diff
@@ -9,9 +9,9 @@
 
 #ifndef __FRC_ROBORIO__
 
+#include <wpi/SmallVector.h>
+
```

Comment:
This include is out of order (wpiformat puts it below memory), see travis failure.